bee-network/bee-gossip/src/init.rs

```diff
@@ -190,8 +190,10 @@ fn init(
         .send(Event::LocalIdCreated { local_id })
         .map_err(|_| Error::LocalIdAnnouncementFailed)?;
 
+    // TODO: rename to PeerStateMap.
```

Comment:
Reminder ✏️

Reply:
Yes, that's something I gonna do with the rewrite of `bee-gossip` I am now working on. Different PR though :slightly_smiling_face: